recipe/meta.yaml

```diff
@@ -11,10 +11,14 @@ source:
   fn: cvxopt-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cvxopt/cvxopt-{{ version }}.tar.gz
   md5: d4ff0a0e51aba93f838a70302a088eea
+  patches:
+    # Fix the environment variable used for SuiteSparse includes.
+    # See this PR ( https://github.com/cvxopt/cvxopt/pull/60 ).
+    - cvxopt_PR_60.diff
```

Comment:
We should replace this by commit ( https://github.com/cvxopt/cvxopt/commit/4e89755cb6aaf6493a49a3ad99dd5d5e007e1a5a ). This can be done by either using part of that commit or the whole commit.